.github/workflows/ci.yaml

```diff
@@ -49,6 +49,12 @@ jobs:
         with:
           run: npm test
 
+      # Run UI tests
+      - name: Run test
+        uses: GabrielBB/xvfb-action@v1.0
+        with:
+          run: npm run ui-test
```

Comment:
This should go into a separate factor in the matrix. At least until the team agrees on how to handle it.

Reply:
I have removed it, please go ahead and set it on your own the way you prefer.

Reply:
done